server/router/activeride.ts

```diff
@@ -8,11 +8,23 @@ const router = express.Router();
 AWS.config.update(config);
 const docClient = new AWS.DynamoDB.DocumentClient();
 
+type ActiveRide = {
+  id: string,
+  startLocation: string,
+  endLocation: string,
+  startTime: number,
+  endTime: number,
+  isScheduled: boolean,
+  riderID: string,
```

Comment:
Should we have an array of riderIDs instead? This will make the management of ride information much more compact and understandable, I think.

Reply:
yeah, i forgot multiple riders could be on the same ride